src/tox/config/loader/str_convert.py

```diff
@@ -63,6 +63,8 @@ def to_command(value: str) -> Command:
                 pos = splitter.instream.tell()
         except ValueError:
             args.append(value[pos:])
+        if len(args) == 0:
+            raise TypeError(f"attempting to parse {value!r} into a command failed")
```

Comment:
Not sure if TypeError is the right error to raise here; what is the output when this happens with tox run? Also needs a changelog entry.

Reply:
> what is the output when this happens with tox run?

The output includes a stacktrace, and the exit code of the program is `2`.

> Not sure if TypeError is the right error to raise here

Yep - good point.  Would `ValueError` be better, and/or did you have something else in mind?

> needs a changelog entry

:+1:

Reply:
Updated the output to produce:

```con
py: failed with unable to determine pip install command: attempting to parse '' into a command failed
```

...with exit code `1` (by handling the `ValueError` and re-raising as a `Fail`)